components/navigation/NavBar.js

```diff
@@ -22,9 +22,10 @@ import {
 import browserLanguageDetector from "../../lib/browserLanguageDetector";
 
 const isMobile = isMobileDevice();
-const uniqueLangs = [...new Set(["EN", "DE"])].map((repo) => ({
-  key: repo,
-  text: repo,
+const uniqueLangs = ["EN", "DE"].map((lang) => ({
```

Comment:
Why the language values are hardcoded here?

Reply:
Also, instead of providing all languages here, we can determine during initial render, which languages are present for this page, using `pathname`. Let's say for some pages, only EN translation is present, then there is no need to show other languages.

Reply:
Thanks. That was an amazing feat. suggestion. Have added the changes 🚀